doordeck-sdk/src/androidMain/kotlin/com/doordeck/multiplatform/sdk/model/responses/SiteResponses.android.kt

```diff
@@ -1,46 +1,52 @@
 package com.doordeck.multiplatform.sdk.model.responses
 
+import com.doordeck.multiplatform.sdk.util.toInstant
+import com.doordeck.multiplatform.sdk.util.toUuid
+import java.time.Instant
+import java.util.UUID
+
 data class SiteResponse(
-    val id: String,
+    val id: UUID,
     val name: String,
     val colour: String,
     val longitude: Double,
     val latitude: Double,
     val radius: Int,
     val passBackground: String,
```

Comment:
Hmm, let's get rid of passBackground as well as colour, it's used together with colour in the current app to make the UI prettier/consistent but the new UI doesn't have it

Reply:
Removed!

Reply:
I requested it back. The website is using it. Michael you probably meant the apps, which are stopping its usage.